include/zephyr/bluetooth/audio/bap.h

```diff
@@ -2239,7 +2239,7 @@ struct bt_bap_broadcast_assistant_mod_src_param {
 	uint8_t num_subgroups;
 
 	/** Pointer to array of subgroups */
-	struct bt_bap_bass_subgroup *subgroups;
+	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
```

Comment:
The changes in this file significantly increase the size of the parameters, and can cause some stack size issues. 

Why is this change required?